eo-maven-plugin/README.md

```diff
@@ -9,19 +9,23 @@ Here is a simple program that gets a year from command line and tells you
 whether it's leap or not:
 
 ```eo
++alias org.eolang.txt.sprintf
++alias org.eolang.io.stdout
++alias org.eolang.txt.text
++alias org.eolang.math.number
```

Comment:
@ivan-egorov42 What do you think about removing these aliases and use `QQ` object instead? e.g. `QQ.math.number`, `QQ.io.stdout` and etc?

Reply:
But QQ like is not working now for sprint?